.sequelizerc

```diff
@@ -0,0 +1,8 @@
+const path  = require('path')
+
+module.exports = {
+    "config": path.resolve('./build/server/database/config', 'config.js'),
+    "models-path": path.resolve('./build/server/database/models'),
+    "seeders-path": path.resolve('./build/server/database/seeders'),
+    "migrations-path": path.resolve('./src/server/database/migrations')
+};
```

Comment:
This isn't linted, but I'm going to trust that the Sequelize itself is sensitive to its own formatting